static/app/components/replaysOnboarding/utils.tsx

```diff
@@ -46,3 +46,14 @@ export const tracePropagationMessage = (
     )}
   </Alert>
 );
+
+export const crashReportCallout = ({link}: {link: string}) => (
+  <Alert type="info" showIcon>
+    {tct(
+      `Interested in receiving feedback only when an error happens? [link:Read the docs] to learn how to set up our crash-report modal.`,
```

Comment:
nice. if someone wants to iterate on the copy writing its just a single file. 

but this paragraph is working for me!